firefox-ios/Client/Frontend/Browser/MainMenu/MainMenuConfigurationUtility.swift

```diff
@@ -619,72 +653,76 @@ struct MainMenuConfigurationUtility: Equatable, FeatureFlaggable {
         with uuid: WindowUUID,
         and configuration: MainMenuTabInfo
     ) -> MenuSection {
-        return MenuSection(
-            options: [
-                configureUserAgentItem(with: uuid, tabInfo: configuration),
-                MenuElement(
-                    title: .MainMenu.ToolsSection.FindInPage,
-                    iconName: Icons.findInPage,
-                    isEnabled: true,
-                    isActive: false,
-                    a11yLabel: .MainMenu.ToolsSection.AccessibilityLabels.FindInPage,
-                    a11yHint: "",
-                    a11yId: AccessibilityIdentifiers.MainMenu.findInPage,
-                    action: {
-                        store.dispatchLegacy(
-                            MainMenuAction(
-                                windowUUID: uuid,
-                                actionType: MainMenuActionType.tapNavigateToDestination,
-                                navigationDestination: MenuNavigationDestination(.findInPage),
-                                telemetryInfo: TelemetryInfo(isHomepage: configuration.isHomepage)
-                            )
+        var options: [MenuElement] = [
+            configureUserAgentItem(with: uuid, tabInfo: configuration),
+            MenuElement(
+                title: .MainMenu.ToolsSection.FindInPage,
+                iconName: Icons.findInPage,
+                isEnabled: true,
+                isActive: false,
+                a11yLabel: .MainMenu.ToolsSection.AccessibilityLabels.FindInPage,
+                a11yHint: "",
```

Comment:
@FilippoZazzeroni this is a scary looking diff for some reason lol but all I am doing is instead of returning the array immedialtely I assign the same array to a var and push the summarizer into it.